src/index.js

```diff
@@ -1,14 +1,23 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import Header from "./header";
+import MainHeroImg from "./mainHeroImg";
+import MainPortfolio from "./mainPortfolio";
+import MainAboutMe from "./mainAboutMe";
+import MainAndMore from "./mainAndMore";
+import Footer from "./footer";
+import * as serviceWorker from "./serviceWorker";
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Header />
+    <MainHeroImg />
+    <MainPortfolio />
+    <MainAboutMe />
+    <MainAndMore />
```

Comment:
```suggestion
    <MainHeroImg />
    <MainPortfolio />
    <MainAboutMe />
    <MainAndMore />
```

Mainコンポーネントの中にセクションとしてこの4つのコンポーネントがあるっていうのでもいいかも。

Reply:
なるほど！変更します。